test/TestUtils.py

```diff
@@ -160,23 +161,34 @@ def test_expand_paths_vars(monkeypatch):
 @pytest.mark.parametrize(
     ('reset_env_var', 'message_prefix'),
     (
-        ('PATH', 'Warning: Failed to locate command:'),
-        ('GIT_DIR', 'Warning: Failed to discover yaml files to lint using git:')
+        ('PATH',
+            "Failed to locate command: "),
+        ('GIT_DIR',
+            "Failed to discover yaml files to lint using git: fatal: not a git repository: ''")
     ),
     ids=('no Git installed', 'outside Git repository'),
 )
 def test_get_yaml_files_git_verbose(
     reset_env_var,
     message_prefix,
     monkeypatch,
-    capsys
+    caplog
 ):
     options = cli.get_config(['-v'])
+    utils.initialize_logger(options.verbosity)
     monkeypatch.setenv(reset_env_var, '')
     utils.get_yaml_files(options)
-    msg1, msg2 = capsys.readouterr().out.splitlines()[:2]
-    assert msg1 == 'Discovering files to lint: git ls-files *.yaml *.yml'
-    assert msg2.startswith(message_prefix)
+
+    expected_info = (
+        "ansiblelint",
+        logging.INFO,
+        'Discovering files to lint: git ls-files *.yaml *.yml')
+
+    assert expected_info in caplog.record_tuples
+    for msg in caplog.messages:
+        if msg.startswith(message_prefix):
+            return
+    pytest.fail('%s not found in logged messages: %s', msg, caplog.messages)
 
 
 @pytest.mark.parametrize(
```

Comment:
@ssbarnea you missed `capsys` here too:

Reply:
In fact I did not miss it, i removed it as it was not used anymore.

Reply:
No, you removed it in the test above, it's a different test that still has it. (`test_get_yaml_files_silent`)

Reply:
In that case there was a bug, wrongly named variable. We still want to check that no output is produced on stderr. The code was functional but it used misleading variable name. I fixed it and also documented the purpose of the test.